calm_ingestor/src/main/scala/uk/ac/wellcome/transformer/controllers/ManagementController.scala

```diff
@@ -4,12 +4,82 @@ import javax.inject.{Inject, Singleton}
 import scala.concurrent.Future
 import scala.concurrent.ExecutionContext.Implicits.global
 
+import java.net.URLDecoder
+import scala.collection.mutable.{Map => MutableMap}
+import scala.xml._
+import scalaj.http._
 import com.twitter.finagle.http.Request
 import com.twitter.finatra.http.Controller
+import scala.util.matching.Regex
 
 @Singleton
 class ManagementController @Inject()() extends Controller {
   get("/management/healthcheck") { request: Request =>
     response.ok.json(Map("message" -> "ok"))
   }
+
+  get("/runfetch") { request: Request =>
+    val data = Http("http://localhost:5000")
```

Comment:
Because the OAI is so slow, I’m using a local server that just serves hard-coded data for now, I haven’t got to multiple pages yet.